PARAMETERS.md

```diff
@@ -22,48 +22,48 @@ DEVICE='cpu'
 
 **USED FOR GRID SEARCH**
```

Comment:
Comments, really???

Reply:
Second place you are allowed to have comments is the documentation haha